made admitted_to mandatory in ConsultationForm

khavinshankar opened the pull request:
Fixes #1743 

**UI before:**
![image](https://user-images.githubusercontent.com/29787772/129432303-af7db3b4-bb2f-4158-9b3a-0299da68b1d3.png)
![image](https://user-images.githubusercontent.com/29787772/129432306-31db7206-3d6d-42aa-a1f7-b3ecd20700fd.png)

**UI now:**
![image](https://user-images.githubusercontent.com/29787772/129432314-b8d45124-b749-452d-97d1-2bc7414a79ec.png)

Removed Admitted (yes or no) field from the form and made Admitted To and Admission Date mandatory fields

nihal467 commented:
![pfjepofw](https://user-images.githubusercontent.com/57055998/129526020-02330748-d98c-4376-89bc-2dd00d507114.PNG)

the alignment changes when there is an error in mobile view

bodhish commented:
@khavinshankar Its was supposed to be in consultation update form. (Log an update form)

nihal467 commented:
![pekfewokf](https://user-images.githubusercontent.com/57055998/129586370-5b4ccbdc-7448-42a2-b63c-760d2f9e0838.PNG)

test failed, the issue was in log update, where the admitted to is mandatory , but without giving any data here, we are able to create a log update

khavinshankar commented:
- [x] reverted changes made in Consultation Form 
- [x] Added mandatory check for admitted to in Log Update Form

![image](https://user-images.githubusercontent.com/29787772/129662609-cf436007-3ba1-41c7-9622-e7fd05bd44cd.png)

nihal467 commented:
test approved
